src/node-saml/types.ts

```diff
@@ -109,6 +109,7 @@ export interface SamlOptions extends Partial<SamlSigningOptions>, MandatorySamlO
   audience?: string;
   scoping?: SamlScopingConfig;
   wantAssertionsSigned?: boolean;
+  maxAssertionAgeMs: number;
```

Comment:
This should be optional since you declare a default in the ctor.

Reply:
I was following the precedent set by `acceptedClockSkewMs` and `requestIdExpirationPeriodMs`: both are non-optional despite declaring defaults in the ctor.

Because part of `ctorOptions`' type is `Partial<SamlOptions>`, I think it's alright to leave these as non-optional. It's expected that some of the `SamlOptions` members will be omitted. Thoughts?